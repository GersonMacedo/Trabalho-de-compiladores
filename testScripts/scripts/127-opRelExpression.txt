! -s SYNTACTIC
! should accept expressions with opRel
program p ;
begin
    a := 0 > 99;
    a := 3 < 54;
    a := 9 = 13;
end .