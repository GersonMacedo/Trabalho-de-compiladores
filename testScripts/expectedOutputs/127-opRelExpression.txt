
Performing up to syntactic analysis at testScripts/scripts/127-opRelExpression.txt
No errors found
